cookbook/src/tx_taproot.md

```diff
@@ -157,7 +151,7 @@ Now we are ready for our main function that will sign a transaction that spends
 # }
 # 
 # fn receivers_address() -> Address {
-#     Address::from_str("bc1p0dq0tzg2r780hldthn5mrznmpxsxc0jux5f20fwj0z3wqxxk6fpqm7q0va")
+#     "bc1p0dq0tzg2r780hldthn5mrznmpxsxc0jux5f20fwj0z3wqxxk6fpqm7q0va".parse::<Address<_>>()
```

Comment:
Is it possible to do a straight `parse::<Address<Network::Bitcoin>>`? That would be less code and more ergonomic in my opinion.